_posts/2020-04-22-#17994.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: pr
+date: 2020-04-22
+title: "Flush undo files after last block write"
+pr: 17994
+authors: [kallewoof]
+components: ["validation"]
+host: vasild
+status: upcoming
+commit:
+---
+
+## Notes
+
+In this Review Club we will dive into how blocks are downloaded and stored on disk. How disk access is arranged and a bug it has caused.
+
+Some concepts that would help to grasp what's going on:
```

Comment:
Suggestion: consider replacing this sentence with a "### Concepts" subheader, e.g.

### Concepts

Reply:
Done, except I used 3 `###`, not 4.